### ISSUE TITLE
GitHub demo is missing IE11 Polyfills

### ISSUE THREAD

**Issue by ghiscoding** [ghiscoding/aurelia-slickgrid#73]
**I'm submitting a bug report**

* **Browser:**
IE 11

**Current behavior:**
Example 3, 4 and 5 are not showing up and are missing some Polyfills. If we take Example 5, we can see this error thrown in the console
```bash
Object doesn't support property or method 'includes'
```

**Expected/desired behavior:**

* **What is the expected behavior?**
No console errors

* **What is the motivation / use case for changing the behavior?**
Support IE 11

**Comment by ghiscoding** [comment]
Did a bit more investigating and we are missing Polyfills in IE11 (not provided by `aurelia-polyfills`) for the following 2 ES6 functions
- `Object.keys` which throws in IE `Object doesn't support property or method 'keys'`
- `String.prototype.includes` which throws `Object doesn't support property or method 'includes'`
   - this can be replaced by `indexOf('x') > -1`

The minimal [aurelia-polyfills] is too minimal and doesn't include polyfills for these 2 errors. 
@jmzagorski 
What do you use as Polyfills for IE11? Did you install extra ones? I can't find how to do that.

If I do a search in the lib, there's about 10 line of codes found for both. I could easily replaced the `includes` but I'm not sure about the `Object.keys`

**Comment by ghiscoding** [comment]
@jmzagorski 
how do you add Polyfill to a WebPack build like the GitHub demo folder?

**Comment by jmzagorski** [comment]
I think all you need to do is add `import 'babel-polyfill'` near the top of `maint.ts`

**Comment by ghiscoding** [comment]
So I've done a few changes to get it working in IE 11.

1. add CDN polyfills to the `index.ejs`
```html
<script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.js" integrity="sha256-qyG2LBYZavhW0RXdL7CQGGv2dX4WP30uIFPguGoOLjE="
    crossorigin="anonymous"></script>
```
2. disabled the Bluebird `longStackTraces` to speed up (**huge difference**)
3. replace all `String.prototype.includes` to `indexOf > -1`

**Comment by ghiscoding** [comment]
closed by PR #78
